### DISCUSSION TITLE
FC1DD4F4-60E5-B4CB-0C0C-8CF030F8C005

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#193] [General]
# FC1DD4F4-60E5-B4CB-0C0C-8CF030F8C005

HD 141569 is a young A-type pre-main sequence star about 362.64 light-years away in the constellation Libra. In 1999, a protoplanetary disk was discovered around the star with a prominent gap between the inner and outer portions of it, leading to speculations that a possible planet was forming in it, though none was found orbiting the star

https://celestia.mobi/addon?item=FC1DD4F4-60E5-B4CB-0C0C-8CF030F8C005

**Comment by SiriusBislifeless** [comment]
Probably the oldest addon in this site
